test/api/models/domain/BusinessIdSpec.scala

```diff
@@ -14,8 +14,17 @@
  * limitations under the License.
  */
 
-package v1.models.request.listAllBusinesses
+package api.models.domain
 
-import api.models.request.RawData
+import support.UnitSpec
 
-case class ListAllBusinessesRawData(nino: String) extends RawData
+class BusinessIdSpec extends UnitSpec {
+
+  "toString" should {
+    "return the BusinessId value" in {
+      val businessId = BusinessId("some id")
```

Comment:
just a minor thing, but to match the convention/pattern, could this be:
```
      val result = BusinessId("some id").toString
      result shouldBe "some id"
```

(or even put BusinessId as a private val at the top, and result = businessId.toString. Or whatevs).

Reply:
(I've put equivalents in bsas-api as you probably saw)